assets/sass/components/global/_googlesitekit-zero-data.scss

```diff
@@ -0,0 +1,85 @@
+/**
+ * CTA styles.
+ *
+ * Site Kit by Google, Copyright 2019 Google LLC
```

Comment:
Please update when copying from another file, although better to setup a snippet so that it's always generated correctly. I can share mine with you if you use VS Code.